### ISSUE TITLE
Duplicate property occured another keys.

### ISSUE THREAD

**Issue by bonegollira** [richlab-corp/eslint-plugin-richlab#11]
```js
  [posV]: 0,
  [posH]: 0
  // => Duplicate property (richlab/css-property-order)
```

**Comment by koozaki** [comment]
@bonegollira Thanks for report. Sorry it's a bug :bow:

I will deal with this issue. Please wait a moment.

**Comment by koozaki** [comment]
@bonegollira 
This bug has fix in published version [1.0.5](https://github.com/richlab-corp/eslint-plugin-richlab/releases/tag/1.0.5)
Thanks for report :smile:
